src/commands/aksInspektorGadget/aksInspektorGadget.ts

```diff
@@ -101,6 +177,68 @@ async function unDeployGadget(
     return await runKubectlGadgetCommands(clustername, command, clusterConfig, kubectl);
 }
 
+async function topTCPGadget(
+    clustername: string,
+    clusterConfig: string,
+    kubectl: k8s.APIAvailable<k8s.KubectlV1>) {
+    const command = "top tcp 30 --all-namespaces --timeout 30";
+
+    return await runKubectlGadgetCommands(clustername, command, clusterConfig, kubectl);
+}
+
+async function topEBPFGadget(
+    clustername: string,
+    clusterConfig: string,
+    kubectl: k8s.APIAvailable<k8s.KubectlV1>) {
+    const command = "top ebpf 30 --all-namespaces --sort cumulruntime --timeout 30";
+
+    return await runKubectlGadgetCommands(clustername, command, clusterConfig, kubectl);
+}
+
+async function topBlockIOGadget(
+    clustername: string,
+    clusterConfig: string,
+    kubectl: k8s.APIAvailable<k8s.KubectlV1>) {
+    const command = "top block-io 30 --all-namespaces --timeout 30";
+
+    return await runKubectlGadgetCommands(clustername, command, clusterConfig, kubectl);
+}
+
+async function topFile(
+    clustername: string,
+    clusterConfig: string,
+    kubectl: k8s.APIAvailable<k8s.KubectlV1>) {
+    const command = "top file 30 --timeout 30";
+
+    return await runKubectlGadgetCommands(clustername, command, clusterConfig, kubectl);
+}
+async function profileCPU(
+    clustername: string,
+    clusterConfig: string,
+    kubectl: k8s.APIAvailable<k8s.KubectlV1>) {
+    const command = "profile cpu --timeout 30";
+
+    return await runKubectlGadgetCommands(clustername, command, clusterConfig, kubectl);
+}
+
+async function snapshotProcess(
+    clustername: string,
+    clusterConfig: string,
+    kubectl: k8s.APIAvailable<k8s.KubectlV1>) {
+    const command = "snapshot process --all-namespaces 30 --timeout 30";
```

Comment:
snapshot gadgets don't require a time or timeout. They will last the time they need to take an snapshot of the processes or sockets in the system. So it should be simply:

```suggestion
    const command = "snapshot process --all-namespaces";
```